runelite-client/src/main/java/net/runelite/client/config/ConfigProfile.java

```diff
@@ -48,6 +50,10 @@ public class ConfigProfile
 	@Getter(AccessLevel.PACKAGE)
 	@Setter(AccessLevel.PACKAGE)
 	private long rev;
+	@Getter
+	@Setter
+	@Nullable
+	private Collection<String> defaultForRsProfiles;
```

Comment:
this should be typed more specifically, as this will sometimes be a LinkedHashSet and sometimes be an ArrayList